lib/cell.rb

```diff
@@ -28,4 +28,20 @@ def fire_upon
     @fired_upon = true
   end
 
+  def render
```

Comment:
I might have some ideas on how to refactor this--not sure how to write that yet, just thinking out loud.